source/includes/steps-install-shell-macos.yaml

```diff
@@ -4,30 +4,48 @@ source:
   ref: navigate-dlc
 ref: navigate-dlc-macos
 ---
```

Comment:
(general comment)
Any chance you can combine steps one and two here. They seem to be repeating a fair bit info information.
https://docs-mongodbcom-staging.corp.mongodb.com/mongodb-shell/docsworker-xlarge/DOCSP-30933-macos-install-instructions/install/#install-from-.zip-file-1